lib/src/model/broadcast/broadcast_controller.dart

```diff
@@ -0,0 +1,126 @@
+import 'dart:async';
+
+import 'package:dartchess/dartchess.dart';
+import 'package:deep_pick/deep_pick.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast.dart';
+import 'package:lichess_mobile/src/model/broadcast/broadcast_repository.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/model/common/http.dart';
+import 'package:lichess_mobile/src/model/common/id.dart';
+import 'package:lichess_mobile/src/model/common/socket.dart';
+import 'package:lichess_mobile/src/utils/json.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'broadcast_controller.g.dart';
+
+@riverpod
+class BroadcastController extends _$BroadcastController {
```

Comment:
Looks like this controller handles all the games of a broadcast round. It would be more clear if it was named `BroadcastRoundController`.